reference-data/rssi-raw-edison/java/segmentation/src/main/com/vmware/herald/calibration/cablecar/segmentation/MovementAnalysis.java

```diff
@@ -102,7 +102,8 @@ public void close() {
 		final Sample distribution = new Sample();
 		movements.forEach(movement -> distribution.add(movement.inertia));
 		// Remove values below mean to discard noise data
-		final double threshold = distribution.mean();
+		final double threshold = distribution.mean() - distribution.standardDeviation();
```

Comment:
Is this for a statistical summary only, or does it mean you're removing some of the original raw data? Having the noise present is useful for machine learning algorithms you see. Thanks

Reply:
Data is retained, just performing statistical analysis here for noise reduction. Removed this from new algorithm as test shows it serves no benefit.